src/main/java/it/gov/pagopa/admissibility/mapper/Initiative2InitiativeConfigMapper.java

```diff
@@ -31,6 +34,7 @@ public InitiativeConfig apply(Initiative2BuildDTO initiative) {
                 .rankingInitiative(initiative.getGeneral().isRankingEnabled())
                 .rankingFields(Boolean.TRUE.equals(initiative.getGeneral().isRankingEnabled()) ? retrieveRankingFieldCodes(automatedCriteriaList) : null)
                 .initiativeRewardType(initiative.getInitiativeRewardType())
+                .isLogoPresent((additionalInfo != null && additionalInfo.getLogoFileName()!= null) ? Boolean.TRUE : Boolean.FALSE)
```

Comment:
simplify this expression (if you want to check if not empty, use StringUtils utility method
```suggestion
                .isLogoPresent(additionalInfo != null && additionalInfo.getLogoFileName()!= null)
```